tactics/class_tactics.ml

```diff
@@ -100,7 +100,7 @@ let _ =
 
 let _ =
   declare_bool_option
-    { optdepr  = false;
+    { optdepr  = true; (* remove in 8.8 *)
```

Comment:
This seems very wrong.  As I understand it, the current default value is the compatibility value, and is more expensive.  At the very least, you should swap the default value at the same time that you deprecate it, and add the setting to the compatibility file.  Alternatively, perhaps swap the default value and add the setting to the compat file, and wait a version before deprecating it.